packages/server/graphql/mutations/messageAllSlackUsers.ts

```diff
@@ -0,0 +1,56 @@
+import {GraphQLNonNull, GraphQLString} from 'graphql'
+import getRethink from '../../database/rethinkDriver'
+import {requireSU} from '../../utils/authorization'
+import MessageAllSlackUsersPayload from '../types/MessageAllSlackUsersPayload'
+import {GQLContext} from '../graphql'
+import SlackServerManager from '../../utils/SlackServerManager'
+import standardError from '../../utils/standardError'
+
+interface MessageSlackUserError {
```

Comment:
I don't like repeating the fields here and in `types/MessageSlackUserError.ts`. I looked at importing `IMessageSlackUserError` but it requires a `__typename`. Creating a `GraphQLInputObjectType` works in this file as we're inputting an error object but we'd need to return a `GraphQLObjectType` in the payload as it's no longer an input. 

One option I looked at was creating a `fields` function and creating both a `GraphQLInputObjectType` and a `GraphQLObjectType`, using the same `fields`. This would reduce replication but add its own complexity of having two different object types.